docs/insiders/faq/sponsoring.md

```diff
@@ -55,6 +55,27 @@ as a collaborator. If your organization doesn't use GitHub or hosts its
 repositories on other platforms, you can mirror the Insiders repository in your
 environment once you have access.
 
+[__We are hosting our repository on Gitlab, can we also use Insiders?__](#github-alternatives){ #github-alternatives }
+
+Absolutely! Material for MkDocs Insiders is designed to be compatible with
+various repository hosting platforms, including GitLab. The key requirement is
+still a GitHub account, as we use GitHub Sponsors for all transactions and
+manage access to the private Insiders repository through GitHub.
+
+Once you've become a sponsor and secured access to the private Insiders
+repository via an individual GitHub account, you can
+[mirror the repository in another location]. This mirroring process not only
+allows for easy integration into your existing workflow but also ensures that
+your projects stay up-to-date with the latest features and improvements of
+Material for MkDocs Insiders.
+
+For any questions about integrating Material for MkDocs Insiders into your
+projects, our [discussion board] is a valuable resource. It offers a space to
+connect with others who may have similar setups, exchange tips, and explore
```

Comment:
This might read better and I think mentioning user needs/requirements makes sense here: "It offers a space to connect with others who may have similar requirements and setups as well as to exchange tips, and explore"

Reply:
Apart from this small comment, this looks good to me. I built it and checked the link. It reads well and I have not spotted any typos or other issues.

Reply:
Thanks for the review and your feedback. I implemented your suggestion and merged it :)